packages/rocketchat-reactions/setReaction.js

```diff
@@ -17,6 +17,10 @@ Meteor.methods({
 			throw new Meteor.Error('error-not-allowed', 'Not allowed', { method: 'setReaction' });
 		}
 
+		if (!RocketChat.emoji.list[reaction] && RocketChat.models.EmojiCustom.findByNameOrAlias(reaction).count() === 0) {
+			throw new Meteor.Error('error-not-allowed', 'Not allowed', { method: 'setReaction' });
```

Comment:
I know this is an older pull request, do you mind changing this to be `error-invalid-emoji` and the text be `Invalid emoji provided.`?